rest/src/test/java/com/carhouse/rest/controllerIT/CarControllerTestIT.java

```diff
@@ -18,11 +18,13 @@
     private static final String HOST = "http://localhost:8086";
     private static final String CAR_LIST_GET_URL = "/carSale/car";
     private static final String CAR_GET_URL = "/carSale/car/";
-    private static final String CAR_ADD_URL = "/carSale/car";
-    private static final String CAR_UPDATE_URL = "/carSale/car";
+    private static final String CAR_ADD_URL = "/carSale/car/";
+    private static final String CAR_UPDATE_URL = "/carSale/car/";
     private static final String CAR_DELETE_URL = "/carSale/car/";
+    private static final String RESPONSE_REGEX = "^\"date\":\"[^\"]*\","
```

Comment:
1. It's an `ERROR_RESPONSE_REGEX`
2. Not sure, but looks like for all `IT` tests the same regex is used. Maybe it might be optimized.

Reply:
Ok